Dockerfile

```diff
@@ -36,6 +34,24 @@ COPY --from=voila_rootfs_builder /etc/ /etc/
 
 
 FROM base_image_python AS kernel_rootfs_builder
+RUN apt-get update && apt-get install -y --no-install-recommends wget
+# Miniconda
+ARG conda_dir="/usr/miniconda3"
+ENV PATH="${conda_dir}/bin:${PATH}"
+ARG PATH="${conda_dir}/bin:${PATH}"
```

Comment:
Isn't this unneeded?

Reply:
Is it bad though? It looks cleaner to me rather than writing the same string rtimes.

Reply:
I understand why you have `ENV PATH`, but why do you need `ARG PATH`, where it's used?

Reply:
Fixed.